src/tangd.c

```diff
@@ -165,10 +189,12 @@ static struct http_dispatch dispatch[] = {
     {}
 };
 
-int
-main(int argc, char *argv[])
+#define DEFAULT_PORT 9090
```

Comment:
Why is default port 9090? If I am not wrong, default port for tang is 80

Reply:
The port 80 is for a web server by convention so using for tangd would be confusing. I used an arbitrary default port that can be documented without overlap with existing services like a web server which are common.